### ISSUE TITLE
SHould bin file allways be created as a result of conversion

### ISSUE THREAD

**Issue by DmitrySemchenok** [ADAPT/ISOv4Plugin#118]
I have noticed that BIN files are created only in case of Raster prescriptions mapping. In case of Vector prescriptions all data is mapped int main xml file. Is this correct or am I missing something in adapt data?

**Comment by knelson-farmbeltnorth** [comment]
@DmitrySemchenok That is correct.   The plugin does not convert vectors to rasters or vice versa.   If you input a vector, it will be written to ISOXML as a polygon prescription.

**Comment by DmitrySemchenok** [comment]
Can you please advise the most optimal way to use plugin in scenario such as this:
We have GeoJosn as an inputh that specifies filed with two points boundary (lower left upper right) and a set of crop zones defined as multipolygons. Each crop zone has an array of products and associated values of planting.
I tried to map zones as Crop zones field and raster prescriptions for values, as well as corp zones filed and vector prescriptions as well as fileld and Vector prescriptions with zones multipolygons mapped to the prescription itself. However each time I got isoxml that cannot be opened by another programm (I tried SMS Basic)

**Comment by knelson-farmbeltnorth** [comment]
@DmitrySemchenok When you configured your multipolygons as a Vector Prescription in ADAPT and exported via the plugin, you will have produced valid ISOXML for a polygon prescription.  However, while polygon prescriptions are valid per the ISO11783-10 specification, there is very limited adoption for them among ISO Task Controllers.
To create a raster prescription via the plugin (which is widely supported), you will first need to rasterize your vector layer.   Both commercial and open source GIS packages have capabilities to convert vectors to rasters.   Since there are several choices to be made by implementers in rasterizing polygons, in addition to the dependency on external GIS tooling, the ISO plugin does not make this conversion.
Once you have your raster layer, you will need to populate the RasterGridPrescription object in the ADAPT model.   Exporting that model should correctly generate a grid prescription in ISOXML.

**Comment by knelson-farmbeltnorth** [comment]
@DmitrySemchenok Take a look at the files in this directory: https://github.com/ADAPT/ISOv4Plugin/blob/develop/TestUtilities/Datacards/

 One or more should have grid prescriptions and if you import via the plugin it will load the ADAPT objects appropriately.

**Comment by DmitrySemchenok** [comment]
@knelson-farmbeltnorth I tried first archive and found there isoxml that contains several prescriptions, however I'm not sure are they raster or vector ones as I tried to export them as both types and plugin did it. However when I tried to export the resulted models back into isoxml plugin failed to do it.

**Comment by gustavobosch** [comment]
Anyone able to create the Grid/Raster prescription using ISOv4Plugin? Does anyone have a working or at least sample code?
